### PR TITLE
Addon update heapster

### PR DESCRIPTION
**Pull request by recollir**
Updated heapster version and added a note about heapster being deprecated.

### CODE REVIEW

**Comment by mikesplain** [comment]
@recollir Looks like you've got a conflict, mind rebasing?

**Comment by recollir** [comment]
@mikesplain rebased. but pull-kops-e2e-kubernetes-aws is flakey again

File: addons/monitoring-standalone/v1.9.0.yaml [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+            limits:
+              cpu: 100m
+              memory: 300Mi
+            requests:
+              cpu: 100m
+              memory: 300Mi
+          command:
+            - /heapster
+            - --source=kubernetes.summary_api:''
+        - image: k8s.gcr.io/addon-resizer:2.1
```
Comment (justinsb) [inline addons/monitoring-standalone/v1.9.0.yaml]: What's the source for this?  I'm trying to make sense of the version of addon-resizer - 2.1 seems older than 1.8.1

**Comment by KashifSaadat** [comment]
Hey @recollir, just wondering if you've had a chance to review the last outstanding comment for this PR in regards to the addon-resizer version?

File: addons/monitoring-standalone/v1.9.0.yaml (hunk 1 above)
Comment (recollir) [reply addons/monitoring-standalone/v1.9.0.yaml]: I can't find the source to this, what appeared to be, "newer" version. I will revert it back to the now 1.8.3

**Comment by recollir** [comment]
I can't find the source to this, what appeared to be, "newer" version. I will revert it back to the now 1.8.3

**Comment by recollir** [comment]
I have now reverted the addon-resizer version to 1.8.3 as that version was used in https://github.com/kubernetes/kubernetes/blob/c65f65cf6aea0f73115a2858a9d63fc2c21e5e3b/cluster/addons/cluster-monitoring/google/heapster-controller.yaml#L82

Sorry for the delay. We had a good long summer in Europe with long vacations and travelling.

**Comment by KashifSaadat** [comment]
Hey @recollir, no problem thanks for making the update :)

The test failures have been fixed in master, you might need to rebase your branch off master for them to disappear.

/retest
/lgtm

**Comment by justinsb** [comment]
Thanks for the version tweaks @recollir 

/approve
/lgtm

Going to force-merge as CI is just upset about the spellchecker and not related to this PR.

**Comment by KashifSaadat** [comment]
Hi @so0k, it looks like the indentation was slightly out in the `addon.yaml` file. I've fixed this in PR #6032.

**Pull request merged**